gc/base/PhysicalSubArenaRegionBased.cpp

```diff
@@ -191,16 +191,44 @@ MM_PhysicalSubArenaRegionBased::doExpandInSubSpace(MM_EnvironmentBase *env, uint
 	MM_HeapRegionManagerTarok *manager = MM_HeapRegionManagerTarok::getHeapRegionManager(_heap);
 	uintptr_t regionSize = manager->getRegionSize();
 
+	/*
+	 due to heap address for each numa node has to alignment with region size and page size, free region count might be not distributed evenly among numa nodes.
+	 use numaNodeCountWithFreeRegions and areFreeRegionsForNode() to inflate all available regions from numa nodes round robin until reaching expanded size.
+	 */
+	uintptr_t numaNodeCountWithFreeRegions = 0;
+#if defined (OMR_GC_VLHGC)
+	numaNodeCountWithFreeRegions = _affinityLeaderCount;
+#	for (uintptr_t idx=0; idx<_affinityLeaderCount; idx++) {
```

Comment:
There is something wrong with this line. See the `#` at the beginning

Reply:
oOps! I have removed '#', Thanks @charliegracie